owner-onboarding-server/src/main.rs

```diff
@@ -394,7 +399,7 @@ async fn main() -> Result<()> {
         // Owner addresses
         owner_addresses,
     });
-
+    let store_path = settings.ownership_voucher_store_config.clone().unwrap().to_string();
```

Comment:
This happens to work with the `Directory` store (also, not after #245), but this is an implementation detail.
You should not ever bypass the Store layer.